virt_backup/backups/pending.py

```diff
@@ -198,9 +198,14 @@ def get_new_tar(self, target, snapshot_date):
         :param target: directory path that will contain our tar. If not exists,
                        will be created.
         """
+        extra_args = {}
         if self.compression not in (None, "tar"):
             mode = "w:{}".format(self.compression)
             extension = "tar.{}".format(self.compression)
+            if self.compression is "xz":
+                extra_args['preset'] = self.compression_lvl
```

Comment:
To be coherent with the rest, could you use `"` instead of `'` please?

Reply:
Of course!
Thanks for the reminder.